qiskit/test/mock/backends/__init__.py

```diff
@@ -21,6 +21,9 @@
 from .yorktown import FakeYorktown
 from .tenerife import FakeTenerife
 from .ourense import FakeOurense
+from .ourense import FakeOurenseFaultyQ1
+from .ourense import FakeOurenseFaultyCX13
+from .ourense import FakeOurenseFaultyCX01
```

Comment:
I would not add these to this directory. Just build them in the tests that you need. Trying to keep this directory a snapshot of real backends (i've been meaning to move them somewhere other than qiskit/test/)

Reply:
Done in 13b92f3af